RustFxA/FxAWebViewController.swift

```diff
@@ -16,8 +16,9 @@ enum DismissType {
 
 enum FxAPageType {
     case emailLoginFlow
-    case signUpFlow
+    case qrCode(url: String)
     case settingsPage
+    case signUpFlow
```

Comment:
this should be removed AFAIK we don't have both flows emailLoginFlow and signUpFlow, there is only one of those flows

Reply:
I believe this may have recently been used by @nishant2718 on master?  I only added it back when I merged his onboarding changes. 

`        // Sign up button closure
        introWelcomeSyncV1Views.signUpClosure = {
            self.didFinishClosure?(self, .signUpFlow)
         }`

`// Sign-up browsing button action
        syncCard.signUp = {
            self.didFinishClosure?(self, .signUpFlow)
        }`

Reply:
let me look... (brb)

Reply:
Yeah those should be changed to .emailLoginFlow, there is no such .signUpFlow anymore, it just shows .emailLoginFlow in both cases.

Reply:
(we have legacy code on the intro screen for two buttons, they both go to the same place now)

Reply:
Got it! Changed :)